### ISSUE TITLE
Basic auth default user

### ISSUE THREAD

**Issue by nemanjadragun92** [kgierke/nuxt-basic-auth#1]
Just to report a bug.

If you use this module and you define

```
  basicAuth: {
    enabled: true,
    users: [
      {
        username: "myUsername",
        password: "myPw",
      },
    ],
    allowedRoutes: ["/api/.*"],
  },
```

It will just append to the users list and still have default value that u have defined as admin:admin

This should not be like that, It should replace your default user array because otherwise, everyone can still access the website by just writing admin:admin instead of this custom-defined password

Thanks

**Comment by kgierke** [comment]
Hi, thanks for reporting that bug, it seems like the default value gets merged with the user value. I gonna remove the default value for admin.

**Comment by kgierke** [comment]
Fixed in: 81b6b1fefb6269babf6d0b02873879f154ee643e
